core/src/main/java/binnie/core/machines/transfer/TransferRequest.java

```diff
@@ -56,7 +57,6 @@ public TransferRequest(final ItemStack toTransfer, final IInventory destination)
 		this.setDestination(destination);
 		this.setTargetSlots(target);
 		this.setTargetTanks(targetTanks);
-		this.transferLiquids = true;
```

Comment:
The value is set repeatedly (a little higher for the first time), while it does not change within the method, so I deleted the line.